torch/nn/functional.py

```diff
@@ -2486,6 +2502,16 @@ def _check_size_scale_factor(dim):
                 and len(scale_factor) != dim:
             raise ValueError('scale_factor shape must match input shape. '
                              'Input is {}D, scale_factor size is {}'.format(dim, len(scale_factor)))
+        if scale_factor is not None and use_scale_factor is None:
```

Comment:
So now that this has been out for a release cycle, what are changes? `use_scale_factor` is set to true ? warnings get removed ? anything else/